docs/source/model_construction.ipynb

```diff
@@ -2321,7 +2321,7 @@
    "source": [
     "plot_2D_pulse((photosphere.signal[0][0], photosphere.signal[1][0]),\n",
     "              x=signal.phases[0],\n",
-    "              shift=signal.shifts,\n",
```

Comment:
`hot['p__phase_shift'], hot['s__phase_shift']` was crashing. 
I'll copy the error message tomorrow.

Reply:
Here's the error when using `signal.shifts`.
Screencapture or text below:
![Screenshot 2022-02-02 at 13 33 13](https://user-images.githubusercontent.com/22171129/152154630-012d73a0-1115-45d9-82ec-7c56a9fa3e26.png)

```
KeyErrorTraceback (most recent call last)
<ipython-input-84-b99d4d04a175> in <module>()
      1 plot_2D_pulse((photosphere.signal[0][0], photosphere.signal[1][0]),
      2               x=signal.phases[0],
----> 3               shift=signal.shifts,
      4               # shift= [ hot['p__phase_shift'], hot['s__phase_shift'] ],
      5               y=signal.energies,

/Users/sebastien/Softwares/anaconda3/envs/xpsi/lib/python2.7/site-packages/xpsi-0.7.9-py2.7-macosx-10.7-x86_64.egg/xpsi/Signal.pyc in shifts(self)
    496     def shifts(self):
    497         """ Returns the hot region phase plus the instrument phase-shift."""
--> 498         return self._shifts + self['phase_shift']
    499 
    500     @shifts.setter

/Users/sebastien/Softwares/anaconda3/envs/xpsi/lib/python2.7/site-packages/xpsi-0.7.9-py2.7-macosx-10.7-x86_64.egg/xpsi/ParameterSubspace.pyc in __getitem__(self, key)
    191 
    192         if isinstance(key, _six.string_types):
--> 193             return self.get_param(key).evaluate(self) # "caller" is subspace
    194 
    195         elif isinstance(key, int):

/Users/sebastien/Softwares/anaconda3/envs/xpsi/lib/python2.7/site-packages/xpsi-0.7.9-py2.7-macosx-10.7-x86_64.egg/xpsi/ParameterSubspace.pyc in get_param(self, name)
    132                         return param
    133 
--> 134             raise KeyError('No parameter in subspace with matching name.')
    135 
    136     def index(self, name):

KeyError: 'No parameter in subspace with matching name.'
```

Reply:
And, the modification `hot['p__phase_shift'], hot['s__phase_shift']` is what is currently in another page of the tutorial (https://xpsi-group.github.io/xpsi/model_construction_streamlined.html), which works. This is why I replaced it.

Reply:
So the issue pops up because the new instrument phase shift parameter was not properly passed. This had to do with a bug raised in #21, fixed by #23. So the fix to the notebook by passing `hot['p__phase_shift'], hot['s__phase_shift']` is not explicitly required anymore.

Reply:
Or if that is easier, could we just accept this pull request as it is and change this line back to the original in the next commit?